src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import duke.commands.Command;
+import duke.data.TaskList;
+import duke.data.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+/**
+ * Driver class
+ */
+public class Duke {
+
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Constructs a new Duke
+     */
+    public Duke() {
+        Storage storage = new Storage("./data/duke.txt");
+        this.ui = new Ui();
+        this.storage = storage;
+        this.tasks = new TaskList(storage.load());
+    }
+
+    /**
+     * Echoes user's input
+     * @param message A message to echo
+     * @return The message user's provided
```

Comment:
You could leave an empty line between description and parameter section as suggested here: https://se-education.org/guides/conventions/java/intermediate.html#comments 
I will just highlight one here 😃